eng/pipelines/aggregate-reports.yml

```diff
@@ -49,6 +49,19 @@ stages:
               Directory: ""
               CheckLinkGuidance: $true
 
+          - task: PowerShell@2
```

Comment:
Let's remove this as we talked in the group chat I don't think it is necessary. We should only add it once find there is a need for it.